src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -303,7 +309,7 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
       key={`field-${p.id}`}
```

Comment:
If this is a Rect key the you can set the key to p.id only as id's should be unique within the same dom-element.

Reply:
Fixed